test/binMaxHeadTest.html

```diff
@@ -178,6 +205,30 @@
     }
 
 var s = null;
+
+describe('Automated Test', function() {
```

Comment:
Add the constraint name and possibly also the definition (you can get this from the comments in the file that defines the function, in this case most likely constraints/binarity.js) to the displayed text of the describe() block.